gradle.properties

```diff
@@ -4,27 +4,27 @@ pluginGroup = com.vk
 pluginName = admstorm
 pluginRepositoryUrl = https://github.com/VKCOM/admstorm
 # SemVer format -> https://semver.org
-pluginVersion = 2.0.6
+pluginVersion = 2.0.7
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 242
-pluginUntilBuild = 242.*
+pluginSinceBuild = 243
+pluginUntilBuild = 243.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = IU
-platformVersion = 2024.2
+platformVersion = 2024.3
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.jetbrains.php:203.4449.22, org.intellij.scala:2023.3.27@EAP
 # PHP Plugin - https://plugins.jetbrains.com/plugin/6610-php/versions
 # Terminal Plugin - https://plugins.jetbrains.com/plugin/13123-terminal/versions
-platformPlugins = com.jetbrains.php:242.20224.427, org.jetbrains.plugins.terminal:242.20224.237
+platformPlugins = com.jetbrains.php:243.21565.211, org.jetbrains.plugins.terminal:243.21565.204
```

Comment:
`org.jetbrains.plugins.terminal` is `platformBundledPlugins`

https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html#ids-of-bundled-plugins

Reply:
Done